### ISSUE TITLE
[BUG] Document skip_imports

### ISSUE THREAD

**Issue by bkreider** [Anaconda-Platform/anaconda-project#378]
#### Description of expected behavior and the observed behavior
I don't think skip_imports is documented.  I found without this setting, a command like: `anaconda-project list-commands` will recursively search ever subdirectory for notebooks and then complain that no command runs that notebook.  This was taking 50s for me on a windows machine with multiple environments.

```
skip_imports:
    notebooks: true
```

**Comment by jbednar** [comment]
Documenting it is good, but also I think it should be false by default. Rarely useful, often problematic.

**Comment by AlbertDeFusco** [comment]
I agree on changing the default. I'll get some time soon to work towards a release.

**Comment by mcg1969** [comment]
Thanks — fair point that this was supported but undocumented.

Documented in #433 alongside the related fix from #343 (which also reduces the volume of output that prompted users to look for `skip_imports` in the first place). I'll close this once that PR lands.

For broader context on the project's direction:

* Anaconda Project has largely been superseded by [pixi](https://pixi.sh/), [uv](https://docs.astral.sh/uv/), and [conda-workspaces](https://github.com/conda-incubator/conda-workspaces).
* See the [Pixi support](https://anaconda-project.readthedocs.io/en/latest/pixi-support.html) page for the `anaconda-project export-pixi` migration tool.
* The `anaconda-project` package itself remains available; existing projects on its stable feature set continue to be supported, but new enhancement work is generally out of scope.
